src/main/java/uk/gov/hmcts/reform/cwrdapi/CaseWorkerRefApiApplication.java

```diff
@@ -24,4 +27,9 @@
     public static void main(final String[] args) {
         SpringApplication.run(CaseWorkerRefApiApplication.class, args);
     }
+
+    @CacheEvict(allEntries = true, cacheNames = "token")
+    public void cacheEvict() {
```

Comment:
spring.cache.cache-names=bearer-token
spring.cache.caffeine.spec=maximumSize=500,expireAfterAccess=60s

we can start with a small value for maximumSize and tweak it based on the number of entries generated.

Reply:
I was meant to remove this method, you're correct it will "automatically take care of evicting cache after ttl" - we'll be able to tweak it based on the QA results 👍

Reply:
You will also have to implement the method to get the value from cache and if not present then fetch by API call
At present, you are only adding it to cache and not reading it from there :)

Reply:
I believe the `@Cacheable` annotation would take care of this?

Reply:
A `maximumSize` of 50 has been added to the cache config here: https://github.com/hmcts/rd-caseworker-ref-api/pull/121/commits/e6f1a1a116b38c21b332dcc4094bb8a48bcb5c37

Reply:
@AdilOoze  - yes thats correct Adil @Cacheable will handle it aptly